### ISSUE TITLE
[FEATURE] Improvement - direct link to my profile when logged in

### ISSUE THREAD

**Issue by amandamartin-dev** [EddieHubCommunity/BioDrop#9786]
### Is this a unique feature?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Is your feature request related to a problem/unavailable functionality? Please describe.

This is an idea to reduce the number of clicks it takes to share from mobile.

Currently to get to my profile, I have to log in -> click my account -> click my profile -> click the share icon

### Proposed Solution

Suggested improvement:
If the user is logged in, present an additional navigation item of EITHER (my profile OR share profile)

The idea here is to decrease friction and improve UX

### Screenshots

In the screenshot below, there could be a direct link to my profile or QR code share to decrease the amount of clicks to share

![mobile-menu-expanded](https://github.com/EddieHubCommunity/BioDrop/assets/97615019/8c0cb800-ecc9-4bbf-9473-ea89586fdf7c)

### Do you want to work on this issue?

No

### If "yes" to above, please explain how you would technically implement this

This is a simple implementation as we already. know if the user is logged in or not, so it would be more about deciding on the UI and where to put the link that is the most discoverable. Would be a good first issue.

**Comment by SaraJaoude** [comment]
Great idea.  What about having this on the dropdown menu using this tailwind component: https://tailwindui.com/components/application-ui/navigation/navbars

Here is a screenshot of the component.  We could have "Share Profile" as one of the options in the dropdown?

<img width="1452" alt="Screenshot 2023-11-19 at 13 56 04" src="https://github.com/EddieHubCommunity/BioDrop/assets/82668196/307ed0fe-af99-40e0-a932-070969644bf1">

**Comment by amandamartin-dev** [comment]
> Great idea. What about having this on the dropdown menu using this tailwind component: https://tailwindui.com/components/application-ui/navigation/navbars
> 
> Here is a screenshot of the component. We could have "Share Profile" as one of the options in the dropdown?
> 
> <img alt="Screenshot 2023-11-19 at 13 56 04" width="1452" src="https://user-images.githubusercontent.com/82668196/284063872-307ed0fe-af99-40e0-a932-070969644bf1.png">

Oh that should be nice! To decrease clicks even further, could be a link to "share profile" maybe?  Not sure which is best, depends on intent

**Comment by eddiejaoude** [comment]
> Oh that should be nice! To decrease clicks even further, could be a link to "share profile" maybe? Not sure which is best, depends on intent

Good idea to add that, also I think we will be moving the share my profile (QR + links) to their own page which will make that easier also

**Comment by amandamartin-dev** [comment]
> > Oh that should be nice! To decrease clicks even further, could be a link to "share profile" maybe? Not sure which is best, depends on intent
> 
> Good idea to add that, also I think we will be moving the share my profile (QR + links) to their own page which will make that easier also

Interesting! Should we change this status to blocked pending that decision or open to dev now?
